### PR TITLE
Added Add nic and Add disks workflows

### PR DESCRIPTION
**Pull request by manjoosh12**
Added Add nic and Add disks workflows

### CODE REVIEW

File: content/io/cloudslang/nutanix/prism/attach_disks_to_vm.sl [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+            - connect_timeout: '${connect_timeout}'
+            - socket_timeout: '${socket_timeout}'
+            - keep_alive: '${keep_alive}'
+            - connections_max_per_route: '${connections_max_per_route}'
+            - connections_max_total: '${connections_max_total}'
+        publish:
+          - vm_name
+          - ip_address
+          - mac_address
+          - power_state
```
Comment (patmante) [inline content/io/cloudslang/nutanix/prism/attach_disks_to_vm.sl:281]: you dont require power_state , mac_adress only vm_disk_uuid and storage_container_id is needed

File: content/io/cloudslang/nutanix/prism/attach_disks_to_vm.sl [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+          - vm_disk_uuid
+          - storage_container_uuid
+          - vm_logical_timestamp
+        navigate:
+          - SUCCESS: success_message
+          - FAILURE: FAILURE
+  outputs:
+    - return_result: '${return_result}'
```
Comment (patmante) [inline content/io/cloudslang/nutanix/prism/attach_disks_to_vm.sl:289]: add vm_disk_uuid as output not exactly return_result

File: content/io/cloudslang/nutanix/prism/attach_disks_to_vm.sl (hunk 1 above)
Comment (manjoosh12) [reply content/io/cloudslang/nutanix/prism/attach_disks_to_vm.sl]: kept only vm_disk_uuid and storage_container_id is needed

File: content/io/cloudslang/nutanix/prism/attach_disks_to_vm.sl (hunk 2 above)
Comment (manjoosh12) [reply content/io/cloudslang/nutanix/prism/attach_disks_to_vm.sl]: added vm_disk_uuid  as output

**Pull request merged**